tests/php/ORM/DataListTest.php

```diff
@@ -1220,6 +1222,107 @@ public function testFilterByNullCase()
         $this->assertSQLNotContains('"DataObjectTest_Fan"."Email" IS NOT NULL', $items9->sql());
     }
 
+    public function testAggregateDBName()
+    {
+    	$filter = new ExactMatchFilter(
+    		'Comments.Count()'
+    	);
+    	$filter->setModel(new DataObjectTest\Team());
+    	$this->assertEquals('COUNT("DataObjectTest_Team"."ID")', $filter->getDBName());
+
+    	foreach(['Comments.Max(ID)', 'Comments.Max( ID )', 'Comments.Max(  ID)'] as $name) {
+	    	$filter = new ExactMatchFilter($name);
+	    	$filter->setModel(new DataObjectTest\Team());
+	    	$this->assertEquals('MAX("DataObjectTest_Team"."ID")', $filter->getDBName());
+
+    	}
+    }
+
+    public function testAggregateFilterExceptions()
+    {
+		$ex = null;
+		try {
+			$filter = new ExactMatchFilter('Comments.Max( This will not parse! )');
+		} catch(\Exception $e) {
+			$ex = $e;
+		}
+		$this->assertInstanceOf(\InvalidArgumentException::class, $e);
+		$this->assertRegExp('/Malformed/', $e->getMessage());
+
+
+		$filter = new ExactMatchFilter('Comments.Max(NonExistentColumn)');
+		$filter->setModel(new DataObjectTest\Team());
+		$ex = null;
+		try {
+			$name = $filter->getDBName();
+		} catch(\Exception $e) {
+			$ex = $e;
+		}
+		$this->assertInstanceOf(\InvalidArgumentException::class, $e);
+		$this->assertRegExp('/Invalid column/', $e->getMessage());
+    }
+
+    public function testAggregateFilters()
+    {    	
+    	$teams = Team::get()->filter('Comments.Count()', 2);
+    	$team1 = $this->objFromFixture(Team::class, 'team1');
+
+    	$this->assertEquals(1, $teams->count());
+    	$this->assertEquals($team1->ID, $teams->first()->ID);
+
+    	$teams = Team::get()->filter('Comments.Count()', [1,2]);
+    	
+    	$this->assertEquals(2, $teams->count());
+    	$this->assertEquals([], array_diff([1,2], $teams->column('ID')));
```

Comment:
Thanks, Dan. I thought about that, but it gets awfully long-winded if you have more than a few records in the list. You need an assertion for every ID. Part of the fun, I guess. Thanks for the tip on Postgres. Did not know that.